pkg/proxy/util.go

```diff
@@ -247,3 +247,15 @@ const (
 	amazonawsCNSuffix = ".amazonaws.com.cn"
 	amazonawsSuffix   = ".amazonaws.com"
 )
+
+// parseBoolValue 解析配置值为布尔类型，支持布尔值和字符串类型
+func parseBoolValue(value any) bool {
+	switch v := value.(type) {
+	case bool:
+		return v
+	case string:
+		return v == "true"
+	default:
+		return false
+	}
+}
```

Comment:
There are no immediate issues found in the provided code, but since it seems to contain specific strings related to AWS (Amazon Web Services), I'd suggest adding `// remove comments` before `const (`. This helps in maintaining code clarity and reduces clutter on the project's line-by-line readability score.

The other parts of your message seem to revolve around parsing boolean values and suggesting improvements for the Go programming language. If you need help with Go-specific coding tasks such as these, feel free to proceed!